app/presenters/provider/admin/account/authentication_providers_show_presenter.rb

```diff
@@ -38,4 +38,8 @@ def authentication_provider_publish_or_unpublish
   def publishing_method
     @authentication_provider.published? ? :delete : :create
   end
+
+  def masked_client_secret
+    @authentication_provider.client_secret.gsub(/./, '*')
```

Comment:
Well, in the developer portal it isn't shown, I'm good with either one of your suggestions.

Reply:
mmm. I think I would just remove the field from the view, but I also don't have a string opinion.

Reply:
Hmm, I personally don't like NOT having the Client Secret field on the "show" view.
The developer portal SSO show view doesn't have Client ID either I think :grimacing: And I find it very confusing too.
It doesn't matter much, because in a future iteration it's expected that the two views ("show" and "edit") would be unified (see [the designs](https://docs.google.com/presentation/d/1HceI3EXlgW8cO01mvcVOG-HoBjenPIGe3RIrm3aa7-g/edit#slide=id.g269e9b010f5_0_116))

But for current PR I'd prefer to keep the field, but hiding it with asterisks (we can use a fixed-length string, e.g. `******` to address @jlledom 's concern about security).

And we also probably need to make sure the designs take into account this JIRA, in order to avoid regression in the future.